server/orders/ordersUtils.js

```diff
@@ -46,6 +46,13 @@ export function addOrder(data)
 
     }
 
+    export function getByMemberId(id)
+    {
+        console.log("member : " + id)
+        return order.find({member_id : id})
```

Comment:
If you search by ID and expect one result to get back, just use findOne and not find that return you an array

Reply:
im searching on orders by member ID, Im expecting to get a list of all the orders of the specific member